sample/src/main/java/org/aerogear/graphqlandroid/data/UserData.kt

```diff
@@ -4,19 +4,33 @@ import android.content.Context
 import android.util.Log
 import com.apollographql.apollo.ApolloCall
 import com.apollographql.apollo.ApolloQueryWatcher
+import com.apollographql.apollo.api.Operation
 import com.apollographql.apollo.api.Response
 import com.apollographql.apollo.cache.normalized.ApolloStore
 import com.apollographql.apollo.exception.ApolloException
 import com.apollographql.apollo.fetcher.ApolloResponseFetchers
 import com.apollographql.apollo.interceptor.ApolloInterceptor
 import org.aerogear.graphqlandroid.*
+import org.aerogear.graphqlandroid.adapter.TaskAdapter
 import org.aerogear.graphqlandroid.model.Task
+import org.aerogear.graphqlandroid.persistence.Mutation
+import org.json.JSONObject
+import java.lang.reflect.Constructor
 import java.util.concurrent.atomic.AtomicReference
+import kotlin.collections.ArrayList
 
 class UserData(val context: Context) {
 
     val noteslist = arrayListOf<Task>()
 
+    val adapter = TaskAdapter(noteslist, context)
+
+    companion object {
+
+        val offlineArrayList =
+            arrayListOf<com.apollographql.apollo.api.Mutation<Operation.Data, Void, Operation.Variables>>()
```

Comment:
Do we need this?
Our implementation with reflection solves both the background and foreground states no?

Reply:
Yeah, don't need this.